pootle/apps/pootle_language/teams.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Permission
+from django.contrib.contenttypes.models import ContentType
+
+from pootle_store.constants import OBSOLETE
+from pootle_store.models import Suggestion, SuggestionStates
+
+
+User = get_user_model()
+
+
+class LanguageTeam(object):
+    roles = dict(
+        member=["suggest"],
+        submitter=["suggest", "translate"],
```

Comment:
I still think that `submitter` doesn't make it obvious what the role is, while `translator` makes it perfectly clear. The same goes for `member` because all roles are members, so I would prefer `suggester` which makes it clear what they can do.